ocrd/setup.py

```diff
@@ -27,6 +27,7 @@
     entry_points={
         'console_scripts': [
             'ocrd=ocrd.cli:cli',
+            'ocrd-dummy-processor=ocrd.cli.dummy_processor:cli',
```

Comment:
You've used `ocrd-dummy` everywhere else (without `-processor`):

```suggestion
            'ocrd-dummy=ocrd.cli.dummy_processor:cli',
```